app/views/sessions/start.html.erb

```diff
@@ -35,6 +35,7 @@
 
       <div class="card-body">
         <%= fh.text_field name: :username_or_email,
+            value: signup_state ? signup_state.suggested_username : '',
```

Comment:
does specifying this value break the form if a user enters a different email but there's an error with it? e.g. the LMS tells us about the email `bob@example.com`, but they think, no my email is `bobby@example.com` and use that, but then we don't find it, we'll clear `bobby@example.com` and replace it with `bob@example.com`.  Can we initialize if not set?